src/prop/theory_proxy.cpp

```diff
@@ -52,11 +52,11 @@ TheoryProxy::TheoryProxy(Env& env,
       d_skdm(skdm),
       d_zll(nullptr)
 {
-  bool trackTopLevelLearned = isOutputOn(OutputTag::LEARNED_LITS);
-  if (trackTopLevelLearned)
-  {
+  // bool trackTopLevelLearned = isOutputOn(OutputTag::LEARNED_LITS);
+  // if (trackTopLevelLearned)
+  // {
     d_zll = std::make_unique<ZeroLevelLearner>(env, propEngine);
```

Comment:
Not relevant for this particular partitioning algorithm, but I want to always collect the learned literals for a different partitioning strategy.

Reply:
Hmm, we shouldn't comment out code in a PR, can we just change the condition of the `if` statement to also include certain partitioning strategies?

Reply:
Good idea. I also might have a totally separate flag for appending the learned literals, but I will put this back for now until I add those partitioning strategies / different flags.